definitions/artifacts/mysql-authentication.json

```diff
@@ -77,9 +77,7 @@
               "type": "string"
             },
             "port": {
-              "type": "integer",
-              "minimum": 0,
-              "maximum": 65535
+              "$ref": "./port.json"
```

Comment:
wonderful

Reply:
ehh actually, I think this needs to be `../types/port.json` but I like the composition!

Reply:
Good catch!